cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/constraint/LastRunConstraint.java

```diff
@@ -30,19 +30,19 @@
  */
 public class LastRunConstraint extends ProtoConstraint.LastRunConstraint implements CheckableConstraint {
 
-  public LastRunConstraint(long millisSinceLastRun) {
-    super(millisSinceLastRun);
+  public LastRunConstraint(long secsSinceLastRun) {
```

Comment:
For the REST API, it could be some fixed time unit, such as seconds, so that the user doesn't need to specify it. But the programmatic one should be taking TimeUnit.